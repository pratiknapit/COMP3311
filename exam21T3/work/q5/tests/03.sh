python3 ./q5 Townhouse 1500000 2 1 0
