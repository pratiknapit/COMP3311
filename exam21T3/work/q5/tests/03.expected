#46052: 44 Hincks Street, Kingsford 2032, 2br, 1ba, 1car, $1252000
#46835: 88 Sturt Street, Kingsford 2032, 2br, 1ba, 0car, $1266000
